### PR TITLE
Split analyzers into C# and VB assemblies

### PR DESCRIPTION
**Pull request by sharwell**
Restores the support that was removed it #626.

### CODE REVIEW

File: src/Microsoft.VisualStudio.Threading.Analyzers.CodeFixes/Microsoft.VisualStudio.Threading.Analyzers.CodeFixes.csproj [hunk 1]
```diff
@@ -34,7 +34,7 @@
   </ItemDefinitionGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="2.8.2" />
+    <PackageReference Include="Microsoft.CodeAnalysis" Version="2.8.2" />
```
Comment (AArnott) [inline src/Microsoft.VisualStudio.Threading.Analyzers.CodeFixes/Microsoft.VisualStudio.Threading.Analyzers.CodeFixes.csproj:37]: Don't code fixes require a reference to Workspaces?

File: src/Microsoft.VisualStudio.Threading.Analyzers.CodeFixes/Microsoft.VisualStudio.Threading.Analyzers.CodeFixes.csproj [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>netstandard1.3</TargetFramework>
```
Comment (AArnott) [inline src/Microsoft.VisualStudio.Threading.Analyzers.CodeFixes/Microsoft.VisualStudio.Threading.Analyzers.CodeFixes.csproj:3]: How is this possible? I thought Roslyn targeted netstandard2.0 nowadays.

File: src/Microsoft.VisualStudio.Threading.Analyzers.CodeFixes/Microsoft.VisualStudio.Threading.Analyzers.CodeFixes.csproj (hunk 1 above)
Comment (sharwell) [reply src/Microsoft.VisualStudio.Threading.Analyzers.CodeFixes/Microsoft.VisualStudio.Threading.Analyzers.CodeFixes.csproj:37]: Microsoft.CodeAnalysis.Common is the root compiler assembly. Microsoft.CodeAnalysis is a higher-level package that includes both languages' compiler packages and workspaces packages.

File: src/Microsoft.VisualStudio.Threading.Analyzers.CodeFixes/Microsoft.VisualStudio.Threading.Analyzers.CodeFixes.csproj (hunk 2 above)
Comment (sharwell) [reply src/Microsoft.VisualStudio.Threading.Analyzers.CodeFixes/Microsoft.VisualStudio.Threading.Analyzers.CodeFixes.csproj:3]: Roslyn 3 targets netstandard2.0. Roslyn 2 targets netstandard1.3. If you want to ensure compatibility with Visual Studio 2017 clients, the build needs to target netstandard1.3.

**Pull request merged**